src/components/PressableWithoutFocus.js

```diff
@@ -23,27 +23,32 @@ const defaultProps = {
 /**
  * This component prevents the tapped element from capturing focus
  */
-class TouchableWithoutFocus extends React.Component {
+class PressableWithoutFocus extends React.Component {
     constructor(props) {
         super(props);
         this.pressAndBlur = this.pressAndBlur.bind(this);
     }
 
     pressAndBlur() {
+        // We need to blur this element when clicked as it opens modal that implements focus-trapping.
+        // When the modal is closed it focuses back to the last active element.
+        // Therefore it shifts the element to bring it back to focus.
+        // https://github.com/Expensify/App/issues/6806
```

Comment:
NAB: @mdneyazahmad please move this comment above the `class PressableWithoutFocus`